### PR TITLE
ft(UTAPI-75): Add metrics for latest checkpoint and snapshot

### PR DESCRIPTION
**Pull request by tmacro**
Also fixes UTAPI-22 so that the latest checkpoint dashboards work correctly

### CODE REVIEW

File: libV2/tasks/CreateCheckpoint.js [hunk 1]
```diff
@@ -55,6 +73,25 @@ class CreateCheckpoint extends BaseTask {
...
+        const resp = await this.withWarp10(async warp10 => warp10.fetch({
+            className: 'utapi.checkpoint.master',
+            labels: {
+                node: warp10.nodeId,
+            },
+            start: 'now',
+            stop: -1,
+        }));
+
+        if (resp.result && (resp.result.length === 0 || resp.result[0] === '' || resp.result[0] === '[]')) {
```
Comment (jonathan-gramain) [inline libV2/tasks/CreateCheckpoint.js]: I believe we should also return `null` if `resp.result` does not exist:

```suggestion
        if (!resp.result || (resp.result.length === 0 || resp.result[0] === '' || resp.result[0] === '[]')) {
```

File: libV2/tasks/CreateSnapshot.js [hunk 2]
```diff
@@ -55,6 +73,25 @@ class CreateSnapshot extends BaseTask {
...
+        const resp = await this.withWarp10(async warp10 => warp10.fetch({
+            className: 'utapi.snapshot.master',
+            labels: {
+                node: warp10.nodeId,
+            },
+            start: 'now',
+            stop: -1,
+        }));
+
+        if (resp.result && (resp.result.length === 0 || resp.result[0] === '' || resp.result[0] === '[]')) {
```
Comment (jonathan-gramain) [inline libV2/tasks/CreateSnapshot.js]: same here

**Review by jonathan-gramain — APPROVED** [review]

**Review by alexanderchan-scality — APPROVED** [review]

**Pull request merged**